<launch>


  <node pkg="spiral_planner" name="spiral_planner" type="spiral_planner" output="screen">

    <param name="S_cost_coef" value="2.0" />
    <param name="Goal_cost_tolerence" value="0.1" />
    <param name="planner" value="dual_path" type="string" />


  </node>

</launch>